rsmtool/test_utils.py

```diff
@@ -22,6 +23,171 @@
 html_warning_regexp = re.compile(r'<div class=".*?output_stderr.*?>')
 section_regexp = re.compile(r'<h2>(.*?)</h2>')
 
+# get the directory containing the tests
+rsmtool_test_dir = Path(__file__).absolute().parent.parent.joinpath('tests')
+
+
+def check_run_experiment(source,
+                         experiment_id,
+                         subgroups=None,
+                         consistency=False,
+                         skll=False,
+                         file_format='csv',
+                         given_test_dir=None):
+
```

Comment:
It might be worth adding docstrings to these `check_run_*()` functions, just so we remember what all of the arguments mean when we are adding new tests in the future. 

I tried this out locally, and it doesn't look like the docstrings would show up in the output of the tests.